cpu/saml21/periph/spi.c

```diff
@@ -97,6 +97,8 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
         case SPI_SPEED_1MHZ:
             f_baud = 1000000;
             break;
+        case SPI_SPEED_4MHZ:
+            return -1;
```

Comment:
Because the code I've been patching already has some case with return -1 inside for unsupported speed (5MHz & 10MHz), so I've decided to stick to the style. But looking at latest code, it looks like the switch/case has been rewritten with a default -1 case, so this hunk should go away.